src/EPR.Payment.Facade.UnitTests/Controllers/Payments/PaymentsControllerTests.cs

```diff
@@ -83,10 +82,14 @@ public void Initialize()
                 It.Is<Func<It.IsAnyType, Exception?, string>>((v, t) => true)),
             Times.Once);
 
-            var contentResult = result as ContentResult;
-            contentResult?.StatusCode.Should().Be(StatusCodes.Status200OK);
-            contentResult?.ContentType.Should().Be("text/html");
-            contentResult?.Content.Should().Contain("window.location.href = 'https://example.com/error'");
+            using (new FluentAssertions.Execution.AssertionScope())
```

Comment:
See my earlier comment on not using the full qualified name in assertionscope. Please do it in all the tets

Reply:
Sure, I will change for all tets.